src/react-components/src/components/01-atoms/Forms/FormDropdown.stories.tsx

```diff
@@ -2,18 +2,17 @@ import * as React from "react";
 
 import FormDropdown from "./FormDropdown";
 import { action } from "@storybook/addon-actions";
-import { withKnobs, text } from "@storybook/addon-knobs";
+import { text } from "@storybook/addon-knobs";
 
 export default {
   title: "FormDropdown",
-  component: FormDropdown,
-  decorators: [withKnobs]
```

Comment:
When the addons are declared when Storybook first starts up, this is not needed.